appyx-interactions/android/src/androidTest/kotlin/com/bumble/appyx/interactions/sample/TestDriveExperimentTest.kt

```diff
@@ -0,0 +1,69 @@
+package com.bumble.appyx.interactions.sample
+
+import androidx.compose.animation.core.LinearEasing
+import androidx.compose.animation.core.tween
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.bumble.appyx.interactions.core.Keyframes
+import com.bumble.appyx.transitionmodel.testdrive.TestDriveModel
+import com.bumble.appyx.transitionmodel.testdrive.interpolator.TestDriveUiModel
+import com.bumble.appyx.transitionmodel.testdrive.interpolator.TestDriveUiModel.Companion.toProps
+import junit.framework.Assert.assertEquals
+import kotlin.math.roundToInt
+import kotlinx.coroutines.runBlocking
+import org.junit.Rule
+import org.junit.Test
+
+class TestDriveExperimentTest {
+
+    @get:Rule
+    val composeTestRule = createComposeRule()
+
+    @OptIn(ExperimentalMaterialApi::class)
+    @Test
+    fun myTest() {
+        composeTestRule.mainClock.autoAdvance = false
+
+        val testAnimationSpec = tween<Float>(durationMillis = 1000, easing = LinearEasing)
+
+        val model = TestDriveModel(NavTarget.Child1)
```

Comment:
I'd say to test the `InteractionModel` rather, as we intend to black box test high level behaviour. (Also on the `InteractionModel` you can ask different operation execution modes).